modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -3,58 +3,74 @@ ifndef::env-site,env-github[]
+You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Text Editor].
 +
 You can also xref:create-api-specification-studio.adoc[create an API specification project in Studio and sync it with Design Center].
-. Publish that API specification to Exchange. +
+. Publish your API specification to Exchange. 
++
 Make your newly designed API available to your entire organization as an Exchange dependency. As you change your API, you can publish new versions of your API specifications to Exchange. See xref:exchange::to-change-raml-version.adoc#best-practices-api-versioning[best practices for API versioning].
 . Import the API specification from Exchange to Studio. +
-You can either xref:import-api-specification-exchange.adoc[import an API specification from Exchange] or xref:import-api-specification-maven.adoc[from Maven]. Studio imports your API specification as a dependency. This means that your Mule application project now holds a reference to the specific API version that you're implementing. Studio monitors your API and notifies you when there is a new version available.
+You can either xref:import-api-specification-exchange.adoc[import an API specification from Exchange] or xref:import-api-specification-maven.adoc[from Maven]. Studio imports your API specification as a dependency. This means that your Mule application project now holds a reference to the specific API version that you’re implementing. Studio checks your API and notifies you when there’s a new version available.
+
+You can use this workflow for OAS and RAML API specifications. It requires Studio 7.8.x or later, and Mule runtime engine 4.1.4 or later.
 
-You can use this workflow for OAS 2.0 and 3.0 and RAML specifications. It requires Studio 7.8.x or later, and Mule runtime engine 4.1.4 or later.
+From Studio 7.18 and later, you can xref:import-api-specification-exchange.adoc[import an AsyncAPI specification from Exchange] and implement it in your Mule project.
 
 == API Specifications as Exchange Dependencies
 
-You can manage APIs for your Mule application as you manage the modules that compose your application. By leveraging Exchange as your unified repository for all your Mule application dependencies, you can monitor the latest versions of your dependencies. This ensures that your application is up-to-date with the latest API requirements.
+Manage APIs for your Mule application as you manage the modules that compose your application. By leveraging Exchange as your unified repository for all your Mule application dependencies, you can check the latest versions of your dependencies. This ensures that your application is up-to-date with the latest API requirements.
```

Comment:
does "as you manage" mean "the same way you manage"?

Reply:
Avoid "This ensures". Suggest 
...dependencies, ensuring that your application is up-to-date with the latest API requirements.

Reply:
OK! Thanks!